config/plugins/interactive_environments/jupyter/config/allowed_images.yml.sample

```diff
@@ -0,0 +1,14 @@
+# This file lists acceptable images to allow runing.
+#
+# This allows you, the admin, to create multiple flavours
+# for your users to run. E.g. maybe you need a geosciences flavour,
+# you can create the image based on our default image and add the
+# appropriate `apt-get/pip install` statements.
+---
+-
+    image: bgruening/docker-jupyter-notebook:16.01
```

Comment:
I haven't had the same experience with dockerhub at all, that's unfortunate that you are.

I'd be happy to have an extra image as a comment. I'm not sure we could meaningfully expose it in a way that Galaxy/GIE framework could automatically fail over.